src/core/fonts.js

```diff
@@ -1759,6 +1759,11 @@ var Font = (function FontClosure() {
         var locaCount = dupFirstEntry ? numGlyphs - 1 : numGlyphs;
         for (i = 0, j = itemSize; i < locaCount; i++, j += itemSize) {
           var endOffset = itemDecode(locaData, j);
+          // The spec says the offsets should be in ascending order, however
+          // some fonts use the offset of 0 to mark a glyph as missing.
+          if (endOffset < startOffset) {
```

Comment:
Just from reading the comment above, I'd sort of expect this condition to read `if (endOffset === 0) {` instead, which seems like it would fix the issues without the regression (although I've not run tests).

Reply:
I was hoping this would be a more general fix, but turns out there are other bad loca tables. In issue1257 it goes 598,74,74